Sources/MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -189,25 +189,35 @@ open class InstructionsCardViewController: UIViewController {
             instructionCollectionView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
         ]
         
-        // Device is in landscape mode and notch (if present) is located on the left side.
-        if UIApplication.shared.statusBarOrientation == .landscapeRight {
-            if UIApplication.shared.userInterfaceLayoutDirection == .rightToLeft {
-                // Language with right-to-left interface layout is used.
-                instructionCollectionViewContraints.append(instructionCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor,
-                                                                                                              constant: 10.0))
+        switch traitCollection.verticalSizeClass {
```

Comment:
Can you please describe this change? Do you think that `.unspecified` should be always equal to `.regular`?

Reply:
I add the `verticalSizeClass` check because the previous check only divide into the `landscapeRight` and other. But in the `portrait/unknown` or the `verticalSizeClass` is `regular/unspecified`, it need to choose the `leadingAnchor` of the `navigationView` to allow the bottom banner and the floating buttons align with it. When it's `.unspecified`, it's better to choose the default as the `.regular` to align with the navigationView layout constraints as well.